apps/svelte.dev/content/blog/2025-01-01-whats-new-in-svelte-january-2025.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "What's new in Svelte: January 2025"
+description: 'Svelte 5 just keeps getting better. Plus, and in-person Svelte Summit!'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+From new syntax to improved dev tooling, the Svelte team has been busy this month building new features in both Svelte and SvelteKit. And as we round the corner into the new year, there's even more to look forward to... like the *10th Svelte Summit*! [Join the Svelte community](https://www.sveltesummit.com/) online or in-person in Barcelona: May 8th and 9th.
+
+Now let's get to the news 👀
+
+## What's new in Svelte
+
+- Snippets can now be exported from module scripts (**5.5.0**, [Docs](https://svelte.dev/docs/svelte/snippet#Exporting-snippets), [#14315](https://github.com/sveltejs/svelte/pull/14315))
```

Comment:
How comfortable are you with deferring to other blogs for content? This list is basically a repeat of the Advent of Svelte blog post, so I'm wondering if an introduction of said events, along with "highlights include..." And then a link to the blog post suffices

Reply:
That's a good idea. I'll think about the best way to integrate that

Reply:
we should probably set up some automation around preventing this, but: best to use root-relative links for anything on svelte.dev

```suggestion
- Snippets can now be exported from module scripts (**5.5.0**, [Docs](/docs/svelte/snippet#Exporting-snippets), [#14315](https://github.com/sveltejs/svelte/pull/14315))
```

Reply:
Automation would be nice because I currently copy and paste from the GitHub preview to do the email version, so keeping these links full URLs would help a lot.